doc/page-features.h

```diff
@@ -151,4 +151,20 @@ foo.userData<Pod>().i;  // 42
 An interface implementation can expose a structure containing some user data. This user data is
 preserved when the structure is copied. Thus it can be made available in a different interface.
 
+\section IPCServerRegistrationFromCpp Registration of an IPC server from C++ code
+Instead of enabling IPC in the QML UI code (with \c IPC.enabled: \c true), it is possible to register an IPC server in the C++ implementation.
```

Comment:
Please limit line length (to 100 in docu).

Reply:
done